### PR TITLE
Attempt pipelines fix

### PR DESCRIPTION
**Pull request by dgdekoning**
A partial fix of the `azure-pipelines.yml` configuration. The [old pull-request](https://github.com/brightway-lca/brightway2-data/pull/2) shows that only the ubuntu 2.7 machine was still failing.

### CODE REVIEW

**Review by cmutel — COMMENTED** [review]
We can delete the 2.7 sections (this is already in the latest pushed version).

The errors with permissions were essentially unsolvable, but disappeared when I deleted and then recreated the pipeline. But currently I need to release a newer version of `bw_processing`, which is blocking this at the moment. I need to write some more tests, and this hasn't been a huge priority, compared to things with actual deadlines, but hopefully that changes now that deliverables have been delivered.

In short: I don't think we need this, but you can edit it if you want to switch to using `coverage` instead of `pytest --cov` (not sure what this does, maybe it is better), and by removing duplicate dependency specs.

**Comment by dgdekoning** [comment]
The switch to `coverage` is not specifically better or worse, and using `pytest --cov` is technically better as it leads to [more consistent behaviour](https://github.com/pytest-dev/pytest-cov#overview).
I'd done most of this work after figuring out that the `python-coveralls` package in the Activity Browser test scripts was actually causing versioning problems (due to breaking changes in `coverage` version 5.0).

You can close this request and give me a poke when you have a newer version of `bw_processing` to go, I'll put up a new one asap :).

**Comment by cmutel** [comment]
@dgdekoning Thanks a lot, this is working now with some inspiration from your fixes
